static/style.css

```diff
@@ -133,6 +133,13 @@ p {
 .footer {
     background-color: var(--primary);
     color: var(--background);
+    bottom:0;
+    left:0;
+    right:0;
+    height: 30px;
+    position: fixed;
```

Comment:
I don't think we want `position: fixed;` because that will make the footer visible even if there's more content outside the current viewport. Ideally, the footer will only show if the user is scrolled down all the way or if there's not a scrollbar.

Take a look at https://css-tricks.com/couple-takes-sticky-footer/ for some inspiration.